src/main/java/com/flansmod/common/guns/raytracing/PlayerHitbox.java

```diff
@@ -176,8 +176,10 @@ public float hitByBullet(EntityBullet bullet, float penetratingPower) {
 
         if (type == EnumHitboxType.HEAD) {
             totalPenetrationResistance = headPenRes;
+        } else if(type == EnumHitboxType.LEGS){
+            totalPenetrationResistance = legsPenRes + feetPenRes;
```

Comment:
change to defaults for legPenRes and feetPenRes?

Reply:
yes

Reply:
how?

Reply:
See line 166, it's a ternary statement to use the armour penRes value OR the default pen res value.

So either this part reverts back to
legsOrBodyOrFeet = legs+feet+chest
head=head

Or the defaults will be changed so:
legs+feet=1
chest=1
head=1
And people's pen res amounts for armour will work differently.